tests/manage/mcg/test_multi_region.py

```diff
@@ -74,8 +73,7 @@ def test_multiregion_bucket_creation(self, mcg_obj, multiregion_mirror_setup):
             len(mirror_attached_pools) == 2
         ), "Multiregion bucket did not have two backingstores attached"
 
-    @tier4
-    @tier4a
+    @tier2
```

Comment:
This test does not fit in tier2; It checks that bucket mirroring policies work by simulating an underlying storage failure which takes one of the backingstores offline. It's related to tier 4.

Reply:
this test indeed involves storage failures but the reason for not keeping it in tier4 is that the storage failure not supposed to cause down time and this storage failure triggers a mechanism that we consider as a product feature

Reply:
According to the tier 4 definition in `OCS Automation by tiers - guidelines`, this seems to fit perfectly into tier 4. Downtime does not constitute a part of the definition. If it's supposed to, I think the document should be updated to reflect that.

The mirroring is being performed and checked regardless of, and before any failure presented -
https://github.com/red-hat-storage/ocs-ci/blob/ef8d8a2af4b092b85320a2dc2335993a71b00cce/tests/manage/mcg/test_multi_region.py#L115-L117
The failure is not required, and is performed purely as a means of achieving deeper, more "real-life" coverage.

I still think that this test fits best into tier 4, but in any other case, 3 would prolly fit better than 2

Reply:
I am ok either way so we can keep it as is. @Shrivaibavi @keesturam what do you think?

Reply:
I am okay to have this in tier4a too.